java/piranha/src/test/resources/com/uber/piranha/XPMethodChainCases.java

```diff
@@ -0,0 +1,94 @@
+package com.uber.piranha;
+
+class XPMethodChainCases {
+
+  interface SomeParam {
+    @BoolParam(key = "other_flag")
+    BoolParameter otherFlag();
+
+    // Annotation and the abstract method should be removed
+    @BoolParam(key = "stale_flag")
```

Comment:
Curious on whether the interface implementation will be removed if `stale_flag` was the only parameter here. It can be a followup and need not be handled as part of this change.

Reply:
Will make a separate PR for this.